base_user_role_history/tests/test_base_user_role_history.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).µ
+
+from datetime import date, timedelta
+
+from odoo.tests.common import SavepointCase
+
+
+class TestBaseUserRoleHistory(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestBaseUserRoleHistory, cls).setUpClass()
```

Comment:
Nice to have: could you add this to avoid some overhead during test execution?
```
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```